src/pages/posts/[postid]/edit.astro

```diff
@@ -0,0 +1,43 @@
+---
+import { EditPostForm } from '../../../components/post-form/post-form';
+import Seo from '../../../components/seo.astro';
+import BaseLayout from '../../../layouts/base-layout.astro';
+import { supabase } from '../../../util/supabase.astro';
+
+const err = (code: number) => {
```

Comment:
This function is duplicated in a bunch of places, can we break it out into a util section?